plugins/modules/pnp_intent.py

```diff
@@ -545,9 +559,10 @@ def get_have(self):
                 function='get_device_list',
                 params={"serial_number": self.want.get("serial_number")}
             )
-            self.log(str(device_response))
+            self.log("Device details of the device with serial number {0} are {1}".format(self.want.get("serial_number"), str(device_response)), "DEBUG")
```

Comment:
self.log("Device details for the device with serial number '{0}': {1}".format(self.want.get("serial_number"), str(device_response)), "DEBUG")

Reply:
Resolved